core/src/Factory.sol

```diff
@@ -103,8 +103,10 @@ contract Factory {
         DEFAULT_RATE_MODEL = defaultRateModel;
     }
 
-    function pause(IUniswapV3Pool pool) external {
-        require(isBorrower[msg.sender]);
+    function pause(IUniswapV3Pool pool, uint40 oracleSeed) external {
```

Comment:
I am still confused about when/how pause is run. Does it have to be called manually? If so, what is stopping someone from not calling it?

Reply:
Anyone can call `pause`. So nothing is stopping anyone from calling it. However, if they call it and the conditions are healthy, then based on [this](https://github.com/aloelabs/aloe-ii/pull/156/files/37333e5bab43db4512330181b965b678d893fc89#diff-fcdc0cc0ffdbed060a1297c96a2c6786bc6ad73d015a0ae3ae18d3841e8362bbR107-R109) I don't think any pausing actually happens.